tscreen.go

```diff
@@ -1398,6 +1398,26 @@ func (t *tScreen) parsePaste(buf *bytes.Buffer, evs *[]Event) bool {
 	return false
 }
 
+func (t *tScreen) parseRegisteredRawSeq(buf *bytes.Buffer, evs *[]Event) bool {
+	b := buf.Bytes()
+
+	if b[0] == '\x1b' && len(t.rawseq) > 0 {
+		strb := string(b)
+		for _, r := range t.rawseq {
+			if strings.HasPrefix(strb, r) {
+				// a registered raw sequence matched the prefix
+				*evs = append(*evs, NewEventRaw(r))
+				t.escbuf.Reset()
+				for i := 0; i < len(r); i++ {
+					buf.ReadByte()
+				}
+				return true
+			}
+		}
+	}
+	return false
+}
```

Comment:
Nit: move it above `parsePaste`?

Reply:
Back from work and the children do more or less sleep, but the PR is already merged.
Hopefully it doesn't bother too much.

Reply:
Nope it doesn't :)